pkg/minikube/command/ssh_runner.go

```diff
@@ -182,13 +196,58 @@ func (s *SSHRunner) Copy(f assets.CopyableFile) error {
 	})
 
 	scp := fmt.Sprintf("sudo mkdir -p %s && sudo scp -t %s", f.GetTargetDir(), f.GetTargetDir())
+	mtime, err := f.GetModTime()
+	if err != nil {
+		glog.Infof("error getting modtime for %s: %v", f.GetAssetName(), err)
+	} else {
+		scp += fmt.Sprintf(" && sudo touch -d \"%s\" %s", mtime.Format(layout), dst)
+	}
 	out, err := sess.CombinedOutput(scp)
 	if err != nil {
 		return fmt.Errorf("%s: %s\noutput: %s", scp, err, out)
 	}
 	return g.Wait()
 }
 
+func (s *SSHRunner) sameFileExists(f assets.CopyableFile, dst string) (bool, error) {
+	// get file size and modtime of the source
+	srcSize := f.GetLength()
+	srcModTime, err := f.GetModTime()
+	if err != nil {
+		return false, err
+	}
+
+	// get file size and modtime of the destination
+	sess, err := s.c.NewSession()
+	if err != nil {
+		return false, err
+	}
+
+	size := fmt.Sprintf("ls -l %s | cut -d \" \" -f5", dst)
+	stat := "stat -c %y" + fmt.Sprintf(" %s", dst)
```

Comment:
Combine these into one command: `stat -c "%s %Y"`

It mean much in the way of performance (~0.006s), but if we're making this change for performance sake, may as well.

Reply:
done